packages/loopback/lib/context/index.ts

```diff
@@ -25,6 +28,25 @@ export class Context {
     return binding;
   }
 
+  bindClass(key: string, ctor: new<T>(...args) => T): Binding {
```

Comment:
Is it just a strongly-typed variant of `bind`? Can we still use `bind` to bind a key to a class?